### ISSUE TITLE
Server crashing when shooting a bow

### ISSUE THREAD

**Issue by Diversion98** [Shadows-of-Fire/Apotheosis#389]
[crash-2021-05-23_19.22.44-server.txt](https://github.com/Shadows-of-Fire/Apotheosis/files/6529888/crash-2021-05-23_19.22.44-server.txt)
If a player shoots a bow (that does a lot of damage), the server might crash, look crash log.
The player also gets a stackoverflowerror.

forge: 36.1.23
apotheosis: 4.6.0

minecraft vanilla bow:

- power 5 
- trueshot 6 
- endless quiver 1
- flame 
- mending
- quick draw 3
- affix for convert to magic damage is dark magic
- sniper affix (damage added = 8.98)

When bow is in hand, the player gains unbreaking 1 due tue astral sorcery.
The player uses dragon arrows but he tested it with vanilla arrows without potion effect as well and the problem persists

Cheers

**Comment by Shadows-of-Fire** [comment]
This crash is coming out of Discord Integration.

**Comment by Diversion98** [comment]
Did some more testing with that said bow.

we deleted discord integration and the server freezes for ever.
He could kill me without no problem (1-shotted me)
if he killed a ghost from the mod Ice and Fire, the server freezes.

So I changed the max tick time to 60000 and we get a crash report because a tick took to long.
Nothing is pointing to any mod. (or I missed something).

Report it to Ice and Fire?
dragons are fine

[crash-2021-05-25_19.54.39-server.txt](https://github.com/Shadows-of-Fire/Apotheosis/files/6541488/crash-2021-05-25_19.54.39-server.txt)
